opencog/cython/opencog/bindlink.pxd

```diff
@@ -7,7 +7,7 @@ cdef extern from "opencog/cython/opencog/BindlinkStub.h" namespace "opencog":
     #   Handle stub_bindlink(AtomSpace*, Handle);
     #
     cdef cHandle c_stub_bindlink "stub_bindlink" (cAtomSpace*, cHandle)
-    cdef cHandle c_execute_atom "do_execute"(cAtomSpace*, cHandle)
+    cdef cHandle c_execute_atom "do_execute"(cAtomSpace*, cHandle) except +
```

Comment:
Does python really need this? In the old days, C++ used to allow you to say "this function throws an exception", and eventually realized this was a bad idea; they mostly ripped it all out. So I'm a little surprised to see the above.

Reply:
"eventually realized" -- it was the C++ standards committee realized it was a bad idea. (and not me)

Reply:
Checked exceptions were poorly implemented in c++, but the idea is not bad by itself. *except +* causes cython to wrap c++ exception with PyErr_SetString call, so it is not c++ throws keyword. Without it throwing an exception from c++ causes SIGABRT

Reply:
OK